frontend/tsconfig.app.json

```diff
@@ -4,7 +4,7 @@
     "env.d.ts",
     "src/**/*",
     "src/**/*.vue"
-  ],
+, "../common/utils"  ],
```

Comment:
There is already a `tsconfig.json` for the files in the `common` directory, this tsconfig should know nothing about files from parent or sibling directories. What problem were you trying to solve here?